pkg/pillar/hypervisor/kvm.go

```diff
@@ -295,9 +295,19 @@ const qemuNetTemplate = `
 `
 
 const qemuPciPassthruTemplate = `
+[device "pci.{{.PCIId}}"]
+  driver = "pcie-root-port"
+  port = "1{{.PCIId}}"
```

Comment:
I would rather use an offset. E.g
```
port = "{{.PCIId + 10}}"
```

Or even better, add the full BDF of the quest device to the `config`

Reply:
Once I find a full documentation of what's recommended here -- I'd love to make it more reasonable -- for now it just what seems to work when tagging a port.